test/examples/TrafficLight.test.js

```diff
@@ -0,0 +1,24 @@
+var assert = require('assert');
+describe("Traffic Light", function() {
+    var include = require('../helpers/Utils').include;
+    include('Core.js');
+    Core.registerEventPoint('DOM_Init'     , {log: false});
+    include('examples/states/trafficLight/Car.js');
+    include('examples/states/trafficLight/TrafficLight.js');
+
```

Comment:
@myfoxtail  okay, I agree, this is not unit tests. I've fixed them according to your suggestions. But let's keep all tests for Car in one file for now. It's more useful, you don't have to switch between files when you read it